panoramapublic/webapp/PanoramaPublic/js/slideshow.js

```diff
@@ -276,8 +264,8 @@ function appendSlidesContainer(parentDivId)
                      '</div>';
 
     parentDiv.innerHTML = slideshowHtml;
-    document.getElementById(nextSlideBtnId).onclick = function() { plusSlides(-1); };
-    document.getElementById(prevSlideBtnId).onclick = function() { plusSlides(1); };
+    document.getElementById(nextSlideBtnId)['onclick'] = function() { plusSlides(-1); };
```

Comment:
This change fine but I don't think should make any difference for CSP purposes. Were you seeing something different?

Reply:
No, just following LabKey convention based on a [comment](https://github.com/LabKey/MacCossLabModules/pull/412#discussion_r1544798161) in PR #412